4001-5000/LIT4315Miracles.xml

```diff
@@ -106,28 +109,119 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+               <label>The sick monk in Dabra Māryām under  ʾAbuna ʾAbsādi</label>
+               <note>Turaiev no. 7</note>
+            </div>
+            <div type="textpart" subtype="chapter" xml:id="DabraLibanos" corresp="LIT6357MEMiracle">
+               <label>A monk from Dabra Libānos, i.e., a follower of ʾAbuna Takla Hāymānot, tells Gabra Bǝrhān, the governor of Goǧǧām (Gwazzān Nagāśi), about the greatness of Mār ʾEwosṭātewos</label>
+               <note>Turaiev no. 8 EMML 1636 2.7</note>
+            </div>
+            
+               <div type="textpart" subtype="chapter" xml:id="Priest" corresp="LIT6358MEMiracle">
+                  <label>A priest monk is saved from his enemy, who wanted to take his office by slandering him to the King</label>
+                  <note>EMML 1636 2.8</note>
+               </div>
+               <div type="textpart" subtype="chapter" xml:id="Rain" corresp="LIT6359MEMiracle">
+                  <label>A man from Axum is saved from heavy rain and thunderstorms</label>
+                  <note>Turaiev no. 9  EMML 1636 2.9</note>
+               </div>
+               <div type="textpart" subtype="chapter" xml:id="Blind" corresp="LIT6360MEMiracle">
+                  <label>On the blind woman from ʾAksum who recovered her sight</label>
+                  <note>Turaiev no. 10</note>
+               </div>
+            
+            
+            <div type="textpart" subtype="chapter" xml:id="Prison" corresp="LIT6361MEMiracle">
+               <label>The abbot who was imprisoned unjustly</label>
+               <note>EMML 1636 2.11</note>
+            </div>
+            <div type="textpart" subtype="chapter" xml:id="Famine" corresp="LIT6362MEMiracle">
+               <label>Miracle of ʾEwosṭātewos: the people of a certain district of Ethiopia who never suffered from drought or famine</label>
+               <note>Turaiev no. 11</note>
+            </div>
+            <div type="textpart" subtype="chapter" xml:id="Sepulchre" corresp="LIT6363MEMiracle">
+               <label>The ruler who tried to take stones from the tomb of the saint for a building</label>
+               <note>Turaiev no. 12</note>
+            </div>
+            <div type="textpart" subtype="chapter" xml:id="Sick" corresp="LIT6365MESick">
+               <label>Miracle of ʾEwosṭātewos: the sick who was healed and on his daughter who had a snake in her womb</label>
+               <note>Turaiev no. 13</note>
+            </div>
+            <div type="textpart" subtype="chapter" xml:id="Slave" corresp="LIT6364MEMiracle">
+               <label>The monk of Dabra Bizan who was taken captive by slave hunters while he was fetching water</label>
+               <note>EMML 1636 2.14</note>
+            </div>
+            <div type="textpart" subtype="chapter" xml:id="Sawa" corresp="LIT6371MEMiracle">
+               <label>Miracle of ʾEwosṭātewos: the woman from Sawa tells her mother to venerate Eustathius</label>
```

Comment:
It is maybe because the title is taken directly from Turaiev, but I find it a bit distracting to read within the same title ʾEwosṭātewos and Eustathius - if both refer to the same Ethiopian saint, it would be legitimate to standardise to ʾEwosṭātewos  everywhere, in my opinion